src/Themes/InputField.css

```
@@ -0,0 +1,13 @@
+.inputfield{
```

Also add a focus state for inputs.
```suggestion
.input-field{
```